account_invoice_mass_sending/models/account_move.py

```diff
@@ -67,3 +70,21 @@ def _send_invoice_individually(self, template=None):
             }
         )
         return wiz.send_and_print_action()
+
+    def _notify_get_recipients(self, message, msg_vals, **kwargs):
+        recipients_data = super()._notify_get_recipients(message, msg_vals, **kwargs)
+        if self.env.context.get("include_followers", False):
```

Comment:
```suggestion
        if not self.env.context.get("exclude_followers", False):
```

Reply:
You need to change how you manage context check in _notify_get_recipients
otherwise you disable notifying followers when there is no context (e.g. in the chatter)

Reply:
(to avoid the " if not condition"  which is less readable than "if condition" you could do the return filtered_recipients_data in the if and the return recipients_data in the else.)